content/sensu-go/5.14/getting-started/learn-sensu.md

```diff
@@ -360,15 +350,15 @@ sensuctl event info sensu-go-sandbox curl_timings --format json | jq .
 ...
 {{< /highlight >}}
 
-Because we configured a metric format, the Sensu agent is able to convert the Graphite-formatted metrics provided by the check command into a set of Sensu-formatted metrics (not shown in the output), which are then sent to the InfluxDB handler that reads Sensu-formatted metrics and converts them to a format InfluxDB accepts.
-Metric support isn't limited to just Graphite; the Sensu agent can extract metrics in multiple line protocol formats, including Nagios performance data.
+Because you configured a metric format, the Sensu agent can convert the Graphite-formatted metrics provided by the check command into a set of Sensu-formatted metrics (not shown in the output). These metrics are then sent to the InfluxDB handler, which reads Sensu-formatted metrics and converts them to a format InfluxDB accepts.
+
+_**NOTE**: Metric support isn't limited to Graphite! The Sensu agent can extract metrics in multiple line protocol formats, including Nagios performance data._
 
-**5. See the HTTP response code events for Nginx in [Grafana](http://localhost:4002/d/go01/sensu-go-sandbox).**
+**5. See the HTTP response code events for Nginx in Grafana.**
 
-Log in to Grafana as username: `admin` and password: `admin`.
-We should see a graph of live HTTP response codes for Nginx.
+Log in to [Grafana](http://localhost:4002/d/go01/sensu-go-sandbox) with username: `admin` and password: `admin`. You should see a graph of live HTTP response codes for Nginx.
```

Comment:
Grafana prompts you to change your password when you follow this link (you can skip this, but it seems kind of weird and disrupts the stream of the training). Any way we can link to a page that's already logged in?

Reply:
We might be able to automate the changing of the password on the backend. I can help out with that int his PR or if you want to get this merged and we can iterate later, that works too!

Reply:
It would be great if we can just have them click a link that opens the Sensu Go sandbox page in Grafana that automatically logs them in...but if we can't do that, it would at least be much better if we can work around the password change issue! I agree this might be better for a follow-up PR.

Reply:
I do believe that Grafana has a read only user that doesn't require a password, so maybe we can leverage that as we aren't having them change anything inside Grafana.

Reply:
I think we can make some changes in the sandbox repo so that a login is not necessary. See https://stackoverflow.com/questions/33111835/how-to-set-up-grafana-so-that-no-password-is-necessary-to-view-dashboards for some relevant discussion.

Reply:
Awesome! Cameron is this something you can help me learn to do? I think I should learn how to do it so I have it in my pocket for future scenarios.